isthmus/src/main/java/io/substrait/isthmus/SubstraitRelNodeConverter.java

```diff
@@ -100,6 +107,57 @@ public RelNode visit(Project project) throws RuntimeException {
     return relBuilder.push(child).project(rexList).build();
   }
 
+  @Override
+  public RelNode visit(Sort sort) throws RuntimeException {
+    RelNode child = sort.getInput().accept(this);
+    List<RelFieldCollation> relFieldCollations =
+        sort.getSortFields().stream().map(sortField -> toRelFieldCollation(sortField)).toList();
+    if (relFieldCollations.isEmpty()) {
```

Comment:
Is this if statement necessary?

Reply:
yes, it is to solve the `select xxx from xxx limit 100` case which does not have the `order by` clause